Google.PowerShell/CloudResourceManager/IamPolicyCmdlets.cs

```diff
@@ -333,4 +350,82 @@ protected override void ProcessRecord()
             }
         }
     }
+
+    /// <summary>
+    /// <para type="synopsis">
+    /// Removes an IAM policy binding to a project.
+    /// </para>
+    /// <para type="description">
+    /// Removes an IAM policy binding to a project. The cmdlet will use the default project if -Project is not used.
+    /// If the binding does not exist, the cmdlet will not raise error.
+    /// </para>
+    /// <example>
+    ///   <code>PS C:\> Remove-GcIamPolicyBinding -Role roles/owner -User abc@google.com -Project "my-project"</code>
+    ///   <para>This command removes the owner role of user abc@google.com in the project "my-project".</para>
+    /// </example>
+    /// <example>
+    ///   <code>PS C:\> Remove-GcIamPolicyBinding -Role roles/container.admin -Group my-group@google.com</code>
+    ///   <para>
+    ///   This command removes the container admin role of the group my-group@google.com in the default project.
+    ///   </para>
+    /// </example>
+    /// <example>
+    ///   <code>
+    ///   PS C:\> Remove-GcIamPolicyBinding -Role roles/container.admin `
+    ///                                  -ServiceAccount service@project.iam.gserviceaccount.com
+    ///   </code>
+    ///   <para>
+    ///   This command removes the container admin role of the serviceaccount service@project.iam.gserviceaccount.com
+    ///   in the default project.
+    ///   </para>
+    /// </example>
+    /// <example>
+    ///   <code>PS C:\> Remove-GcIamPolicyBinding -Role roles/editor -Domain example.com</code>
+    ///   <para>This command removes the editor role of all users of the domain example.com in the default project.</para>
+    /// </example>
+    /// <para type="link" uri="(https://cloud.google.com/iam/docs/overview)">
+    /// [Google Cloud IAM]
+    /// </para>
+    /// </summary>
+    [Cmdlet(VerbsCommon.Remove, "GcIamPolicyBinding", DefaultParameterSetName = ParameterSetNames.User)]
+    public class RemoveGcIamPolicyBindingCmdlet : AddOrRemoveGcIamPolicyBindingCmdlet
+    {
+        protected override void ProcessRecord()
+        {
+            string role = GetRole();
+            string member = GetMember();
+
+            // Remove the role from existing bindings.
+            ProjectsResource.GetIamPolicyRequest getRequest = Service.Projects.GetIamPolicy(new GetIamPolicyRequest(), $"{Project}");
+            Policy existingPolicy = getRequest.Execute();
+            bool needToExecuteRequest = false;
+
+            foreach (Binding binding in existingPolicy.Bindings)
+            {
+                if (string.Equals(role, binding.Role, StringComparison.OrdinalIgnoreCase))
+                {
+                    if (binding.Members.Contains(member, StringComparer.OrdinalIgnoreCase))
```

Comment:
This will have problems in weird cases like "smith@gmail.com" and "chrissmith@gmail.com". Is there a reason why we aren't just using `Equals`?